SentryTestUtils/Invocations.swift

```diff
@@ -42,8 +42,8 @@ public class Invocations<T> {
     }
     
     public func record(_ invocation: T) {
-        queue.async(flags: .barrier) {
-            self._invocations.append(invocation)
+        queue.async(flags: .barrier) { [weak self] in
```

Comment:
`m`: That seems like a fix that could be in an extra PR, so the whole test suite immediately benefits from this improvement.